sbt-dumpschema-plugin/src/main/scala/org/virtuslab/akkasaferserializer/DumpSchemaPlugin.scala

```diff
@@ -0,0 +1,24 @@
+package org.virtuslab.akkasaferserializer
+
+import scala.tools.nsc.Global
+import scala.tools.nsc.plugins.{Plugin, PluginComponent}
+
+class DumpSchemaPlugin(override val global: Global) extends Plugin {
```

Comment:
Hmmm maybe for the sake of clarity... pls rename to `DumpSchemaCompilerPlugin` since there's also an sbt plugin over there

Reply:
Yeah, it's very confusing now... there's now an object of the same FQCN, but in different sbt module :///

Reply:
I'd also add some package structure under `org.virtuslab.akkasaferserializer` to discern classes more easily

Reply:
Could you elaborate on the last point further?

Reply:
I'd add subpackages under `org.virtuslab.akkasaferserializer`, so that each standalone component (serializability-checker compiler plugin, dump-event-schema compiler plugin, sbt plugin) has its own package... or a similar division

Reply:
Note: this can be done on a follow-up PR ofc

Reply:
Let's move renaming stuff to #48